cookbook-react/README.md

```diff
@@ -0,0 +1,27 @@
+# Components
+
+This branch contains all the components the Cookbook App will be rendering and displaying.
```

Comment:
```suggestion
This branch contains all the React components the Cookbook App will be rendering and displaying.
```

Reply:
Oh, we'll need to change this README. Add a `TODO` please

Reply:
I've created a card for this and the other pending things to do.